diagnostic.md

```diff
@@ -45,7 +62,23 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+```CSS
+.big {
+  ...
+}
+
+#contentPane {
+
+}
 
+.important, .red {
```

Comment:
This will target any element that has `.important` AND any element that has `.red`. We want elements that have BOTH classes: `.important.red`